cli/src/command-processor.ts

```diff
@@ -3,7 +3,7 @@ import yargs from 'yargs/yargs';
 import * as readline from 'readline';
 import {CommandMessage, READY_FOR_MESSAGE, CHECK_READY} from './commands';
 import {ExecuteMessage, IdentifyMessage, ScanMessage} from './commands';
-import {UDPScanConfig} from '../radio/radio-controller';
+import {UDPScanConfig} from 'local-home-testing/build/src/radio';
```

Comment:
Do they need to come from `/build` or can the values be pulled directly from the Typescript source?

Reply:
Since the TypeScript compiler doesn't resolve the `local-home-testing` local dependency, I believe this import gets resolved at runtime, so I can't target the typescript source.  One option is using `tsconfig` path mapping to hard code the TypeScript source but I'm unsure that would be much better so will leave as-is for now.